recipe/meta.yaml

```diff
@@ -31,16 +29,20 @@ test:
     - pip check
   requires:
     - pip
-    - python <3.10
+    - python
```

Comment:
don't think python is required for pip check

Reply:
Correct. `python <3.10` was an old trick for python 3.10 build-out and must be removed
```suggestion
```